components/css/buckyless/notifications.less

```diff
@@ -37,33 +37,48 @@
         background-color: rgba(0, 0, 0, 0.12);
       }
 
+      .notifications-list__action {
+        min-width: 60px;
+        text-align: right;
+      }
+
       notifications-list-item {
         width: 100%;
-      }
 
-      .notification-buttons {
-        display: flex;
-        margin: 0 8px;
+        .md-list-item-text {
+          padding: 16px 0;
+
+          .notification__priority-message {
+            color: @grayscale8;
+            font-size: 12px;
+            margin-bottom: 6px;
+
+            .material-icons {
+              min-height: 18px;
+              min-width: 18px;
+              height: 18px;
+              width: 18px;
+              font-size: 18px;
```

Comment:
:question: `18px` is getting repeated a lot, is this something that can/should come from a variable?